src/uct/api/uct.h

```diff
@@ -1495,6 +1509,22 @@ typedef struct uct_md_mem_attr {
      * to uct_md_mem_query is returned as is.
      */
     size_t            alloc_length;
+
+    /**
+     * Dmabuf file descriptor to expose memory regions across devices. Refer
+     * (https://01.org/linuxgraphics/gfx-docs/drm/driver-api/dma-buf.html).
+     * If the md does not support querying fd object associated with the region,
+     * then UCT_DMABUF_FD_INVALID is returned. It is responsibility of the
+     * user to close returned fd once used using close().
+     */
+    int               dmabuf_fd;
+
+    /**
+     * Offset of given address from the start of the memory object (identified
+     * by dmabuf_fd) backing the memory region being queried. This offset may
+     * not be used if UCT_DMABUF_FD_INVALID is retured as dmabuf_fd.
```

Comment:
1. what is the default value if not set? 0 ?
2. returned from where? the last sentence is not clear